controller/book.controller.js

```diff
@@ -1,14 +1,48 @@
-
-
+import prisma from "../lib/prisma.js";
 /**
- * 
- * @param {import("express").Request} req 
- * @param {import("express").Response} res 
+ *
+ * @param {import("express").Request} req
+ * @param {import("express").Response} res
+ * @description get all books information
  */
-export async function getAllBooks(req, res){
-    try {
+export async function getAllBooks(req, res) {
+  let { page_count, limit } = req.query;
+
+  if (limit > 20) {
+    return res.status(401).json({
+      success: false,
+      message: "you can only request 20 records per page",
+    });
+  }
+
+  // if no page_count and limit query is provided
+  // default value for limit is 20
```

Comment:
What's wrong with your grammar?

Reply:
You can make the code better...but will merge, hoping to see the code in a better state next time.